src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,24 @@
 import React from 'react';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
+
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[]
+};
+
+export const MoviesList: React.FC<Props> = ({ movies = [] }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <div key={movie.imdbId} className="card" data-cy="Movie">
```

Comment:
```suggestion
      <div 
        key={movie.imdbId}
        className="card" 
        data-cy="Movie"
      >
```

Reply:
Same rules)

Reply:
will be more clear to move .card block to MovieCard component. Make it like here
`{movies.map(item => (
      <MovieCard key={item.imdbId} movie={item} />
    ))}`